examples/with-babel-macros/README.md

```diff
@@ -27,5 +27,5 @@ plugin, then we can use the `preval.macro` in `pages/index.js` to pre-evaluate
 code at build-time. `preval.macro` is effectively transforming our code, but
 we didn't have to configure it to make that happen!
 
-Specifically what we're doing is we're prevaling the username of the user who
+Specifically what we're doing is we're prevailing the username of the user who
```

Comment:
This is referring to the ` "preval.macro"` dependency, as in pre-evaluating, so it should not be changed.

Reply:
Makes sense! Thanks for pointing it out.

Reply:
Fixed on https://github.com/vercel/next.js/pull/20516/commits/f617d7b8daf590c657e03f15b13d9ace5ef9b4e7